### PR TITLE
First Map

### PR DESCRIPTION
**Pull request by Mokumokuren00**
# First Map
## Scenes
- Created a basic template of the house.
## Art additions
- Void & Wall tile (both placeholders).
- PC sprite (placeholder).
## Code changes
- changed **_move_and_collide()_** to **_move_and_slide()_**.
## Collisions
- Added collisions to the temporary wall tiles.
- Changed to a smaller cylindrical collision shape for the PC.
## Other
- Started writing a README file with the project's folder structure.
- Moved Isi's art to "Art" > "Sprite Ideas".
- Removed Alex's test folder (hopefully).

### CODE REVIEW

File: Prefabs/Player.tscn [hunk 1]
```diff
@@ -1,9 +1,11 @@
 [gd_scene load_steps=4 format=2]
 
-[ext_resource path="res://icon.png" type="Texture" id=1]
+[ext_resource path="res://Art/Sprites/pc#placeholder.png" type="Texture" id=1]
 [ext_resource path="res://Scripts/Player.gd" type="Script" id=2]
 
-[sub_resource type="RectangleShape2D" id=1]
+[sub_resource type="CapsuleShape2D" id=1]
```
Comment (jsalex7) [inline Prefabs/Player.tscn:6]: good

File: Scripts/Player.gd [hunk 2]
```diff
@@ -17,5 +17,5 @@ func _physics_process(delta):
 	else:
 		velocity = velocity.move_toward(Vector2.ZERO, FRICTION * delta)
 	
-	move_and_collide(velocity * delta);
+	move_and_slide(velocity);
```
Comment (jsalex7) [inline Scripts/Player.gd:20]: why? Do we need sliding at collision?

**Review by jsalex7 — COMMENTED** [review]
Nice work, the sliding is questionable

File: Scripts/Player.gd (hunk 2 above)
Comment (Mokumokuren00) [reply Scripts/Player.gd:20]: So, both work pretty much the same, but:
move_and_collide has this little quirk where, when you collide with an object, your player sticks to that object and it gets harder to move away.
move_and_slide on the other hand, doesn't do that.
I hope this answer is acceptable.

**Pull request merged**